packages/perennial/contracts/multiinvoker/MultiInvoker.sol

```diff
@@ -160,7 +165,7 @@ contract MultiInvoker is IMultiInvoker, UInitializable {
      * @param product Product to deposit funds for
      * @param amount Amount of DSU to deposit into the collateral account
      */
-    function depositTo(address account, IProduct product, UFixed18 amount) private {
+    function depositTo(address account, IProduct product, UFixed18 amount) internal {
```

Comment:
can you make all these `internal` function start with `_` (they should have to begin with)